src/sideEffect/saga/crudResponse.js

```diff
@@ -23,20 +23,20 @@ import linkToRecord from '../../util/linkToRecord';
 function* handleResponse({ type, requestPayload, error, payload }) {
     switch (type) {
     case CRUD_UPDATE_SUCCESS:
-        return yield [
+        return requestPayload.redirect ? yield [
```

Comment:
I think you should show the notification even if the redirect is false. Same for the other cases.